<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectFile">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToServer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Соединиться с сервером</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addrInput">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>70</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
